network-api/networkapi/management/commands/ensure_related_posts.py

```diff
@@ -0,0 +1,30 @@
+from django.core.management.base import BaseCommand
+
+from networkapi.wagtailpages.models import BlogPage
+
+
+class Command(BaseCommand):
+    help = '''
+        Ensures that all blog posts have three explicit related posts,
+        rather than relying on the "find related posts for a blog post"
+        function every time the blog page is requested by a user.
+    '''
+
+    def handle(self, *args, **options):
+        all_posts = BlogPage.objects.all()
+        total_posts = all_posts.count()
+
+        print('Total number of posts to update:', total_posts)
+
+        for i, post in enumerate(all_posts):
+            print(
+                f"Processing post {i+1} out of {total_posts}, "
+                "{post.related_posts.all().count()} preexisting related posts"
+            )
+
+            post.save_revision()
+
+            print(f'-  post {i+1} updated to {post.related_posts.all().count()} related posts')
```

Comment:
This f-string is acting strange. Terminal shows me:
```
Processing post 23 out of 537, {post.related_posts.all().count()} preexisting related posts
                                       ^ Not formatted 
```

Reply:
wow okay that one's weird.

Reply:
```suggestion
            total_related_posts= post.related_posts.all().count()
            print(f'-  post {i+1} updated to {total_related_posts} related posts')
```